### PR TITLE
kubeadm: code clean up about kustomize

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
While reading and learning the source code, find some problems and fix them. :)

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cmd/kubeadm/app/util/kustomize/kustomize.go [hunk 1]
```diff
@@ -207,7 +208,7 @@ func (km *Manager) Kustomize(data []byte) ([]byte, error) {
 		return nil, err
 	}
 	name := "resource.yaml"
-	_ = memFS.WriteFile(filepath.Join(fakeDir, name), b)
+	memFS.WriteFile(filepath.Join(fakeDir, name), b)
```
Comment (SataQiu) [inline cmd/kubeadm/app/util/kustomize/kustomize.go]: I searched the code base, and it seemed that the return values can be ignored.
e.g. https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cli-runtime/pkg/kustomize/builder_test.go#L136

**Review by yastij — COMMENTED** [review]
/lgtm
/assign @fabriziopandini

**Review by fabriziopandini — APPROVED** [review]
@SataQiu thanks!
/approve
/lgtm

**Pull request merged**